pennylane_lightning_gpu/src/simulator/StateVectorCudaMPI.hpp

```diff
@@ -64,6 +64,25 @@ extern void setBasisState_CUDA(cuDoubleComplex *sv, cuDoubleComplex &value,
                                const size_t index, bool async,
                                cudaStream_t stream_id);
 
+template <class Precision, class index_type> class CSRMatrix {
```

Comment:
Also, can we add a simple docstring for this class.

Reply:
Done!